lib/active_model/serializer/adapter/attributes.rb

```diff
@@ -2,42 +2,73 @@ module ActiveModel
   class Serializer
     module Adapter
       class Attributes < Base
+        attr_accessor :_nested_associations
+
+        def initialize(serializer, options = {})
+          options ||= {}
+
+          self._nested_associations = ActiveModel::Serializer::Utils
+            .include_args_to_hash(options[:include])
+
+          super(serializer, options)
```

Comment:
So, once we have the option passed in, I'd only use it in the initializer if there were some reason for using it there.  Otherwise, I'd use it when needed